roles/osp_observability/templates/prometheus.yml.j2

```diff
@@ -1,18 +1,34 @@
-#jinja2: trim_blocks:False
 global:
   scrape_interval: {{ prometheus_scrape_interval }}
   external_labels:
-    {% if ansible_fqdn %}agent_host: {{ ansible_fqdn }}{% endif %}
-    {% if openstack_cloud_name %}cloud_name: {{ openstack_cloud_name }}{% endif %}
+    {% if ansible_fqdn -%}
```

Comment:
The hyphen in `-%}` is just for whitespace control to not show empty lines on the `if` and `endif` lines if I remember correctly?

Reply:
It is for whitespace control and removes all _trailing_ whitespace (including line breaks) after the tag. This positions "agent host" at the same indent level that the "if" is at.  This is again improved upon in #19

Reply:
Thanks for clarifying!